lambdas/service/app.py

```diff
@@ -619,7 +641,15 @@ def start_manifest_generation():
     retry_url = f'{protocol}://{base_url}{endpoint_path}'
 
     try:
-        return manifest_service.get_manifest_status(token, retry_url)
+        status_code, retry_after, location = manifest_service.get_manifest_status(token, retry_url, browser_request)
```

Comment:
I think it's fine if the service method returns the JSON rather than a tuple with the JSON being generated here.